### PR TITLE
Add Scalastyle checks.

### PR DESCRIPTION
**Pull request by brkyvz**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 addSbtPlugin("me.lessis" % "bintray-sbt" % "0.2.1")
 
 libraryDependencies += "org.scala-sbt" % "scripted-plugin" % sbtVersion.value
+
+addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.6.0")
```
Comment (ahirreddy) [inline project/plugins.sbt]: Why not use 0.7.0?

**Comment by ahirreddy** [comment]
Left just 1 comment, otherwise LGTM!

File: project/plugins.sbt (hunk 1 above)
Comment (brkyvz) [inline project/plugins.sbt]: didn't know it existed :) I'll make it 0.7

**Comment by brkyvz** [comment]
@ahirreddy Thanks a lot!!!

**Pull request merged**